src/servers/main.cc

```diff
@@ -223,7 +224,16 @@ std::vector<Option> options_{
      "Reserve a portion of GPU memory for TensorFlow models. Default "
      "value 0.0 indicates that TensorFlow should dynamically allocate "
      "memory as needed. Value of 1.0 indicates that TensorFlow should "
-     "allocate all of GPU memory."}};
+     "allocate all of GPU memory."},
+    {OPTION_TF_ADD_VGPU, "tf-add-vgpu",
+     "Add a tensorflow virtual GPU instances on a physical GPU. Input "
+     "should be 2 integers and 1 float separated by semicolons in the format "
+     "<physical GPU>;<number of virtual GPUs>;<memory limit per VGPU>. "
```

Comment:
Memory limit should be specified in bytes? Mbytes? I think Mbytes is best but you should document.